src/helpers/date_to_string.js

```diff
@@ -0,0 +1,5 @@
+const DateToString = (date) => {
+  return date.toDateString();
```

Comment:
We don't need this file since we have a helper function in the server.js file called DateToString